### DISCUSSION TITLE
replacement of voltage regulators

### DISCUSSION THREAD

**Discussion by PPZ-00** [BlueSCSI/BlueSCSI-v2#86] [Build it Yourself]
Hello, there's shortage of AMS1117-2.85 at JLCPCB. Could I use the 3.3V instead?

**Comment by androda** [comment]
I recommend against using a 3.3v regulator here. Only 2.8 or 2.85v are acceptable for U3.

You have the option to "preorder" parts at JLCPCB, so anything missing can be acquired before placing an assembly order.

**Reply by PPZ-00** [reply]
I tried to preorder, but it seems that jlcpcb is not finding anything at usual stores (Mouser etc.). 
Actually they answered to me that they can preorder it for me, at minimum quantity of 63 pieces, and when I did order with price of 0.12€ per piece, they later asked another 78€, claiming that price went from 0.12 to 1.2-1.3€... 
Anyway, I will order the PCBs without voltage regulators, and get them from another source.
Thank you for answer.

**Comment by androda** [comment]
> I tried to preorder, but it seems that jlcpcb is not finding anything at usual stores (Mouser etc.). Actually they answered to me that they can preorder it for me, at minimum quantity of 63 pieces, and when I did order with price of 0.12€ per piece, they later asked another 78€, claiming that price went from 0.12 to 1.2-1.3€... Anyway, I will order the PCBs without voltage regulators, and get them from another source. Thank you for answer.

An alternate part is the [TLV1117LV28DCYR](https://www.digikey.com/en/products/detail/texas-instruments/TLV1117LV28DCYR/3479418).  This one is more modern but has a narrower input voltage range.  The AMS1117 supports up to something like 12 volts input and the TLV part only supports up to 5.5v max.  Fortunately we are running off the 5v power rails so the TLV part is fine.

**Reply by PPZ-00** [reply]
Yes, I got it from the BOM of db25, as it offered straight away the TLV1117LV28CYR so I have chosen that one instead of AMS... 
The only thing I am trying to figure out now is why JLCPCB is charging me for assembly of internal 50 pin double the money than for db25.

**Comment by androda** [comment]
> Yes, I got it from the BOM of db25, as it offered straight away the TLV1117LV28CYR so I have chosen that one instead of AMS... The only thing I am trying to figure out now is why JLCPCB is charging me for assembly of internal 50 pin double the money than for db25.

The Desktop model has a lot more "extended parts" than the DB25 does.  JLCPCB loves to upcharge for just about everything.  Desktop model simply costs more to make.

**Reply by PPZ-00** [reply]
Yeah, I got it hahaha. Anyway, great project, and thank you for your help! .

I ordered all of them with TLV voltage regulator finally and everything went ok.
